reactivesocket-client/src/main/java/io/reactivesocket/client/LoadBalancer.java

```diff
@@ -151,22 +250,29 @@ private synchronized void refreshAperture() {
 
         long now = Clock.now();
         boolean underRateLimit = now - lastApertureRefresh > APERTURE_REFRESH_PERIOD;
```

Comment:
Yes, it limits the number of aperture refresh, to 1 every 15 seconds to be resilient to small spikes.